packages/pages/src/dev/dev.ts

```diff
@@ -23,7 +23,6 @@ const handler = async ({
   noInit,
   scope,
   noGenFeatures,
-  noGenTestData,
```

Comment:
Can we still use this flag to disable the call in the server?

Reply:
added an alias so you can use --local or --noGenTestData and it won't run the generate command